src/test/java/io/fabric8/maven/docker/config/handler/property/PropertyConfigHandlerTest.java

```diff
@@ -940,6 +947,16 @@ void testResolve() {
         validateRunConfiguration(resolved.getRunConfiguration());
     }
 
+    @Test
+    void testResolveAssembly() {
+
+    }
```

Comment:
Is this PR a WIP? If yes, please mark it as a draft.

Reply:
It is not. Just missed that, will remove it 👍

Reply:
It is not, I have removed the empty tests 👍